### PR TITLE
fix(desk-tool): add full timestamp tooltip to timeline items

### PR DESCRIPTION
**Pull request by sjelfull**
### Description

This adds the full change timestamp to a tooltip when you hover over the relative time stamp in the version timeline.

https://user-images.githubusercontent.com/10508/190409236-557b0d86-1115-448e-b9ed-32e36a080bad.mp4

Closes #3251

### What to review

- Is this the best UI solution?

### Notes for release

- Added full timestamp to version timeline

### CODE REVIEW

**Comment by rexxars** [comment]
Supersedes https://github.com/sanity-io/sanity/pull/3251?

**Comment by sjelfull** [comment]
> Supersedes #3251?

Correct, this is more aligned with how we show timestamps elsewhere. Will update the #3251.

File: packages/@sanity/desk-tool/src/panes/document/timeline/timelineItem.tsx [hunk 1]
```diff
@@ -98,3 +118,8 @@ export function TimelineItem(props: {
     </Root>
   )
 }
+
+const TimestampBox = styled(Box)`
```
Comment (RitaDias) [inline packages/@sanity/desk-tool/src/panes/document/timeline/timelineItem.tsx:122]: Move this to the styled file in ln10 :)

**Review by RitaDias — CHANGES_REQUESTED** [review]
Small comment for structure but everything else looks good ✨

**Comment by robinpyon** [comment]
Thanks @sjelfull, sorry this was stuck in limbo. This been ported to V3 with a few minor tweaks in #4100

**Pull request closed without merging**